paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapter.kt

```diff
@@ -30,18 +30,11 @@ interface CustomerAdapter {
      */
     val canCreateSetupIntents: Boolean
 
-    /**
-     * A list of payment method types to display to the customer.
-     * Valid values include: "card", "us_bank_account"
-     * If null or empty, the SDK will dynamically determine the payment methods using your Stripe Dashboard settings.
-     */
-    val paymentMethodTypes: List<String>?
-
     /**
      * Retrieves a list of payment methods attached to a customer
      * @return a list of [PaymentMethod]s.
      */
-    suspend fun retrievePaymentMethods(): Result<List<PaymentMethod>>
+    suspend fun retrievePaymentMethods(paymentMethodTypes: List<String>): Result<List<PaymentMethod>>
```

Comment:
Makes `CustomerSheet` the single source of truth for this rather than split it between `CustomerAdapter` and `CustomerSheet`.